hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1603,6 +1603,13 @@ public enum OperationStatusCode {
    */
   public final static boolean HBASE_SERVER_USEIP_ENABLED_DEFAULT = false;
 
+  /**
+   * If reading block cost time more than the threshold, a warning will be logged.
+   */
+  public static final String FS_READER_WARN_TIME = "hbase.fs.reader.warn.time";
```

Comment:
Better add this somewhere in the hbase-server module?

Reply:
there are no constants for hbase-server module, i refer "hbase.writer.unified.encoded.blocksize.ratio" in HFileWriterImpl class, and define it in org.apache.hadoop.hbase.io.hfile.HFileBlock.FSReaderImpl.